includes/admin/settings/register-settings.php

```diff
@@ -905,7 +905,7 @@ function edd_get_registered_settings() {
 						'id'    => 'debug_mode',
 						'name'  => __( 'Debug Mode', 'easy-digital-downloads' ),
 						'check' => __( 'Enabled',    'easy-digital-downloads' ),
-						'desc'  => sprintf( __( 'Check this to enable logging of certain behaviors to a file. %s', 'easy-digital-downloads' ), $debug_log_link ),
+						'desc'  => sprintf( __( 'Check this box to enable Debug Mode. %s', 'easy-digital-downloads' ), $debug_log_link ),
```

Comment:
I feel like tying the link into the translatable string is unnecessary and something we could go ahead and change now. What would you think of removing the `sprintf` and just appending the link to the string? It feels like less work for translators.